code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,10 +14,10 @@
 	var/shapeshift_type
 	var/list/current_shapes = list()
 	var/list/current_casters = list()
-	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,\
-		/mob/living/simple_animal/pet/dog/corgi,\
-		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
-		/mob/living/simple_animal/bot/ed209,\
+	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/pet/dog/corgi,
+		/mob/living/simple_animal/hostile/carp/ranged/chaos,
+//		/mob/living/simple_animal/bot/ed209,
```

Comment:
Just dropping wiz as a gamemode entirely would be my suggestion when it's phrased like that. Make it adminbus only until we get a sufficient population. Or raise the population required for it. Either works.